cpp/src/parquet/properties.h

```diff
@@ -176,59 +176,100 @@ class PARQUET_EXPORT WriterProperties {
       return this;
     }
 
+    /**
+     * Enable dictionary encoding in general for all columns. Default enabled.
+     */
```

Comment:
Thanks for doing this!

In general we use `/// Doc comment` syntax over the Javadoc style `/** */`, if you don't mind keeping it consistent

Reply:
I used /** */ style because the ```encoding``` function also uses this style. I can change them too to make it consistent

Reply:
Ah, indeed…but other functions in this file use `///` (e.g. `compression_level`). Sorry for the confusion there.